task2/script.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+function isArrayOne(arr) {
+  return arr.constructor === Array;
+}
+
+function isArrayTwo(arr) {
+  return arr instanceof Array;
+}
+
+function isArrayThree(arr) {
+  return {}.toString.call(arr) === '[object Array]'
```

Comment:
it works for me in chrome 56
https://learn.javascript.ru/class-instanceof
don't work in console. try "( {}.toString.call([]) )"